manifests/params.pp

```diff
@@ -38,8 +38,8 @@
   $framework_config = {
     'framework.server.name'     => $::fqdn,
     'framework.server.hostname' => $::fqdn,
-    'framework.server.port'     => '4440',
```

Comment:
I mean if they have disabled ssl but not bothered to also specify the correct framework_config then it should still use sensible  (non-ssl) defaults.